.rubocop.yml

```diff
@@ -1,18 +1,21 @@
 AllCops:
   TargetRubyVersion: 2.3
+  TargetRailsVersion: 4.2
   Include:
     - '**/config.ru'
     - '**/Rakefile'
     - '**/*.rake'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
-    - 'script/**/*'
     - 'vendor/**/*'
 
 Style/StringLiterals:
   EnforcedStyle: double_quotes
 
+Style/FrozenStringLiteralComment:
```

Comment:
Seems like this should be enabled / was enabled during the lint. There's an `always` option, or `when_needed` is the default ([ref](https://rubocop.readthedocs.io/en/latest/cops_style/#stylefrozenstringliteralcomment)).

Reply:
I might have run 'rake lint' before updating .rubocop.yml 
I'll open another PR for this.

Reply:
Oh, I meant—is there any reason we shouldn't enable this cop? Freezing strings apparently [improves performance slightly](https://bugs.ruby-lang.org/issues/8976#note-30) and will be the [default behavior in Ruby 3](https://bugs.ruby-lang.org/issues/11473).